open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/services/SubjectAreaCategoryRESTServices.java

```diff
@@ -346,4 +335,36 @@ public SubjectAreaCategoryRESTServices() {
         }
         return response;
     }
+
+    /**
+     * Get Category terms
```

Comment:
This should have a description. Get the terms that are categorized by this Category.